src/main/java/seedu/duke/model/gates/NandGate.java

```diff
@@ -9,9 +9,10 @@
      * Performs bitwise NAND (~&) operation on the input instance variables.
```

Comment:
These JavaDoc could be improved, since the `boolean` is returned also.

Reply:
oh! thanks for pointing out